code/datums/diseases/transformation.dm

```diff
@@ -228,3 +228,27 @@
 		if(4)
 			if (prob(20))
 				affected_mob.say(pick("Bark!", "AUUUUUU"))
+
+
+/datum/disease/transformation/cake
+	name = "Cake Hat Infection"
+	cure_text = "Take off the damn hat"
+	agent = "Dixbfloppin"

```

Comment:
Transcending history, and the world, a tale of souls and swords, eternally retold.